packages/create-nx-workspace/bin/create-nx-workspace.ts

```diff
@@ -206,16 +207,23 @@ function showHelp() {
 
 async function getConfiguration(parsedArgs) {
   try {
+    let style, appName, preset;
+
     const name = await determineWorkspaceName(parsedArgs);
-    const preset = await determinePreset(parsedArgs);
-    const appName = await determineAppName(preset, parsedArgs);
-    const style = await determineStyle(preset, parsedArgs);
+    const thirdPartyPackage = await determineThirdPartyPackage(parsedArgs);
```

Comment:
```
let preset = await determineThirdPartyPackage(parsedArgs);
```

Then 

```
if (!preset) {
  preset = await determinePreset(parsedArgs);
  // ...
}
```

Then we can leave the return as before. :)

One less variable and branch condition to consider.

Reply:
done!